spec/features/h3_object_creation_spec.rb

```diff
@@ -118,10 +119,11 @@
     end
 
     find('.nav-link', text: 'Deposit', exact_text: true).click
-    click_link_or_button 'Deposit', class: 'btn-primary'
 
-    #   # if you have previously agreed to the terms within the last year, there will be no checkbox
-    #   check('I agree to the SDR Terms of Deposit') if page.has_css?('#work_agree_to_terms', wait: 0)
+    # if you have previously agreed to the terms within the last year, there will be no checkbox
```

Comment:
`# if you have ever agreed to the terms ...`

Reply:
```suggestion
    # if you have ever agreed to the terms, there will be no checkbox
```